docs/modules/edged.md

```diff
@@ -0,0 +1,61 @@
+# EdgeD
+
+## Overview
+
+EdgeD is an edge node module which manages the POD lifecycle. With help of the POD specifications provided as JSON from Cloud via MetaManager, it ensures the containers for the POD's (with containers as described in POD spec) are launched over the edge node, are running and healthy.
```

Comment:
it ensures the containers for the POD's (with containers as described in POD spec) are launched over the edge node, are running and healthy.
Can this description be a little shorter ? Also , it sounds a bit repetitive.